airflow/configuration.py

```diff
@@ -258,6 +259,21 @@ class AirflowConfigParser(ConfigParser):
         ("scheduler", "parsing_cleanup_interval"): ("scheduler", "deactivate_stale_dags_interval", "2.5.0"),
     }
 
+    # A mapping of new section -> (old section, since_version).
+    deprecated_sections: dict[str, tuple[str, str]] = {"kubernetes_executor": ("kubernetes", "2.4.2")}
```

Comment:
The problem is from here, I guess, because according to this [PR](https://github.com/apache/airflow/pull/26873), `kubernetes_executor` is the new config while `kubernetes` is the old

Reply:
Ops. Understood the comment now :(

Reply:
But comment on L262 doesn't match L263

```
new section -> (old section, since_version)
kubernetes -> ("kubernetes_executor", "2.4.2")
```